infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/Warnings.java

```diff
@@ -40,5 +40,16 @@
       COMMAND_IS_CONFIGURED_IN_PLUGIN_WITH_MULTIPLY_CONTAINERS_WARNING_MESSAGE_FMT =
           "There are configured commands for plugin '%s' that has multiply containers. Commands will be configured to be run in first container";
 
+  public static final int JSON_IS_NOT_A_VALID_REPRESENTATION_FOR_AN_OBJECT_OF_TYPE_WARNING_CODE =
+      4108;
+  public static final String JSON_IS_NOT_A_VALID_REPRESENTATION_FOR_AN_OBJECT_OF_TYPE_WARNING_FMT =
+      "Json is not a valid representation for an object of type: '%s'";
```

Comment:
Please reformulate it to make it more user closer. From a user's perspective of view, it tells him nothing why and when it happened, what are consequences and how to fix it.

Reply:
Messages updated.